docs/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/use-new-nodes-in-an-infra-provider/vsphere/shutdown-vm.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Graceful Shutdown for VMware vSphere Virtual Machines
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/use-new-nodes-in-an-infra-provider/vsphere/shutdown-vm"/>
+</head>
+
+In Rancher v2.8.3 and later, you can configure the graceful shutdown of virtual machines (VMs) for clusters hosted on VMware vSphere infrastructure. Graceful shutdown introduces a delay before the VM is forcibly deleted, which allows time for terminating any running processes and open connections.
+
+In RKE2/K3s, you can set up graceful shutdown when you create the cluster, or edit the cluster configuration to add it afterward. 
+
+In RKE, you can edit node templates to similar results.
+
+## Set Up Graceful Shutdown During VMware vSphere Cluster Creation 
+
+<Tabs>
+<TabItem value="RKE2/K3s">
+
+In RKE2/K3s, you can configure new VMware vSphere clusters with graceful shutdown for VMs. 
+
+1. Click **☰ > Cluster Management**.
+1. Click **Import Existing** to add your cluster, or click **Create** and select **VMware vSphere** to provision a new cluster.
```

Comment:
In the past Rancher wasn't able to distinguish whether an imported cluster is vSphere or not. Did that change?